lib/config/options/index.ts

```diff
@@ -2856,6 +2856,14 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'cloneSubmodulesFilter',
+    description:
+      'List of submodules names or patterns to clone when cloneSubmodules=true.',
+    type: 'array',
+    subType: 'string',
+    default: ['*'],
```

Comment:
`*`  only matches `something` but not `some/sub/module`
```suggestion
    default: ['**'],
```